app/main.py

```diff
@@ -1,8 +1,31 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.wife = None
+        self.husband = None
+        self.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    new_list = [Person(p["name"], p["age"]) for p in people]
+    for person_id in range(len(new_list)):
```

Comment:
zwróć uwagę, że `people` to lista dictów, więc nie musisz używać indexów, tylko odnosić się bezpośrednio do konkretnej osoby, czyli 
```suggestion
    for person in people:
```